app/Exceptions/Handler.php

```diff
@@ -90,6 +90,29 @@ private static function isOAuthSessionException(Throwable $e): bool
             && $e->getMessage() === 'Authorization request was not present in the session.';
     }
 
+    private static function modelNotFoundMessage(ModelNotFoundException $e): string
+    {
+        $ids = $e->getIds();
+
+        $model = $e->getModel();
+        $modelTransKey = "models.name.{$model}";
+
+        $params = [
+            'model' => trans_exists($modelTransKey, $GLOBALS['cfg']['app']['fallback_locale'])
+                ? osu_trans($modelTransKey)
+                : trim(strtr($model, ['App\Models\\' => '']), '\\'),
+        ];
+
+        if (empty($ids)) {
+            $key = 'models.not_found_no_ids';
+        } else {
+            $key = 'models.not_found';
+            $params['ids'] = implode(', ', $ids);
```

Comment:
I wonder if the id shouldn't be visible. I took it from the original laravel exception message

Reply:
I'm thinking it shouldn't be shown in general, but in the context of the other issue, it'd make sense to know which one is missing? 🤔